### ISSUE TITLE
[Tab]: Tabs have focus trap

### ISSUE THREAD

**Issue by janhassel** [carbon-design-system/carbon#5148]
## Environment

> Operating system
macOS 10.14.6

> Browser
Chrome 79

## Detailed description

> What version of the Carbon Design System are you using?
`carbon-components-react@7.9.2`
`carbon-components@10.9.2`

> What did you expect to happen?
When having the browser focus on a tab component, pressing <kbd>⇥ Tab</kbd> again should lead to the next item in the tab order.

> What happened instead?
Focus is trapped

## Steps to reproduce the issue

1. Open https://codesandbox.io/s/sharp-hill-zi261
2. Click a tab to set keyboard focus there
3. Try pressing <kbd>⇥ Tab</kbd>
4. Click on the second button
5. Pressing <kbd>⇧ Shift</kbd> <kbd>⇥ Tab</kbd> focuses on first button
6. Pressing <kbd>⇧ Shift</kbd> <kbd>⇥ Tab</kbd> again focuses on tab
7. Focus is trapped again.

> Please create a reduced test case in CodeSandbox
https://codesandbox.io/s/sharp-hill-zi261

**Comment by emyarod** [comment]
this has been addressed in #5124 which will be in an upcoming release

related https://github.com/carbon-design-system/carbon/pull/4784#issuecomment-576879328

**Comment by janhassel** [comment]
Awesome, thanks. I tried searching for related issues but clearly wasn't looking hard enough.

What is the status about the `@nightly` release channel? The last nightly build available on npm is about 2 months old. It'd be great to have this tracking master to see if an issue has already been addressed and will be released soon.
cc @joshblack
